### DISCUSSION TITLE
LMAO I love this one

### DISCUSSION THREAD

**Discussion by HuyHung1408** [Futura-Py/FluxCalc#41] [General]
![image](https://user-images.githubusercontent.com/86362423/170906661-bd39d6cb-92c4-4b69-8ee1-1b9117067188.png)
 
Love how the 'development' word is in the main content 😂

**Comment by lg-nineoneninethreenine** [comment]
Try rickrolling if they try the indev feature, that would be lolrofllmao.

**Reply by not-nef** [reply]
What do you mean with indev feature?

**Reply by HuyHung1408** [reply]
In development ig

**Reply by not-nef** [reply]
Ah ok, can implement

**Reply by lg-nineoneninethreenine** [reply]
Ah, and also, IM BAAAACKKKK!

Ever wondered why there was no response from me for the almost entirety of last week?

**Reply by lg-nineoneninethreenine** [reply]
I want to do the rickroll prank indev thingy. Maybe as an undercover thing? Let's talk about it on the team chat on GitHub. Is it fine if I make a separate team for adding funny, yet cool things like this? I swear that I will not add a prank or Easter egg without explaining it to y'all and getting a go-ahead with a 10-4.

Sorry for not being online for the last week, I was in a summer camp, and the funny yet angsty thing is that school started a day before travel, I left my phone here due to my sibling's "Shall I switch the phone off and leave it here?". I was regularly checking GitHub from there via browsers, but didn't login since it would ask MFA codes/perms to my then shutdown phone, so was offline for the week, but I'm back. Don't worry, it was nice, wish you guys were there too, we could have been a gang.

**Reply by HuyHung1408** [reply]
hi y'all, welcome back 😎

**Reply by not-nef** [reply]
hi man, good to see you back! 👋

**Reply by not-nef** [reply]
i can do the rickroll, its one line of code

**Reply by HuyHung1408** [reply]
seems good

**Reply by lg-nineoneninethreenine** [reply]
> hi man, good to see you back! 👋 

That was a nice "welcome back!" But the funny bit is that I doubt the gangsters in my class(seriously, they are _the flipping same_) would greet me the same way.

Maybe when I'm old enough to reveal my identity, we could meet in the same camp. Same place. I won't reveal it now, since that's an identity threat, but just a hint: it's in India.

**Reply by lg-nineoneninethreenine** [reply]
> hi y'all, welcome back 😎

Thanks, that was good too! Feels just like a regular day now. Man, I got my yearly achievement list now, thanks to you guys. You made me think of some ideas(during the camp), hope I achieve them this year.

**Reply by not-nef** [reply]
I doubt that we could meet somewhere (although i would love to) because i live 6.7k kilometers (air) away from india lmao

**Reply by lg-nineoneninethreenine** [reply]
I'll try coming after my graduation(I hope I do).

**Reply by not-nef** [reply]
You want to go to another country?

**Reply by sumeshir26** [reply]
Hey @im-coder-lg Welcome back!!

**Reply by lg-nineoneninethreenine** [reply]
> Hey @im-coder-lg Welcome back!!

Thank you guys! Your welcome was better than my classmates' tbh. They acted like nothing happened and they never noticed me. But anyway, I have plans to destroy them in what I am good at(you guys helped me become good at this too, so... thanks 🙂 a bunch) as well as some more apps and games.

> You want to go to another country?

Well, not like immigration, like a tourist visit. I like India since it is PERFECT for me, like from the climate to my needs, it is SO good here. But when I have a chance, I'll try visiting.

**Reply by not-nef** [reply]
yeah it would be very cool if we could meet someday when im done with school. for me its still 5 years. when will you graduate?

**Reply by lg-nineoneninethreenine** [reply]
Well, next year, it's Class 10, so... I don't have a plan lol. Perks of being me.

**Reply by not-nef** [reply]
How many classes do you have left? Next year is class 9 for me

**Reply by sumeshir26** [reply]
Im in class 9 now, 2023-24 is class 10 for me. That means boards in india

**Reply by HuyHung1408** [reply]
same

**Reply by lg-nineoneninethreenine** [reply]
> How many classes do you have left?

Well, there's the CBSE system that has Class 10 to 12, then college for another few years. So I doubt we would meet when we are 18. It's not about the passport, it's about the travel and all that. Surely I wouldnt be left alone. Maybe we'll try this: we will meet at the camp I went to when my identity is out. Sorry for speaking like a spy, but one of my prized possessions on the Internet is my identity. I liket o remain in te dark, until I wish to come out. But the upside is that I'll do it someday during college.

> Next year is class 9 for me

Wait, so your schooling was delayed or what(honestly a question, I swear)? I thought we all were class 9..

**Reply by not-nef** [reply]
I think we have a different school system. I am still in the schoolyear 21/22 (a schoolyear here starts in september and ends in july) so i am still in 8th grade. Also i have school until 13th grade.

**Reply by not-nef** [reply]
And you all live in india i assume so its easier for you guys to meet

**Reply by HuyHung1408** [reply]
me Vietnam 🇻🇳

**Reply by HuyHung1408** [reply]
> I think we have a different school system. I am still in the schoolyear 21/22 (a schoolyear here starts in september and ends in july) so i am still in 8th grade. Also i have 13 grades.

It's the same here in vn. No 13 grades ofc

**Reply by HuyHung1408** [reply]
> I think we have a different school system. I am still in the schoolyear 21/22 (a schoolyear here starts in september and ends in july) so i am still in 8th grade. Also i have 13 grades.

It's the same here in vn. But no 13 grades
